src/app/core/store/user/user.actions.ts

```diff
@@ -0,0 +1,6 @@
+import { createAction, props } from '@ngrx/store';
+
+export const signIn = createAction(
+  '[User] User signIn',
+  props<{ role: 'guest' | 'user' }>()
```

Comment:
We have 'manager' role too

Reply:
Yes, there is a mistake, Admin is missing. 
I took this part of the code from Sasha's PR, I will replace it before the merge.

Reply:
I alredy replace this on guest| user| manager

Reply:
Done